GitVersion.yml

```diff
@@ -1,5 +1,8 @@
-mode: "ContinuousDeployment"
+mode: ContinuousDeployment
 branches:
   master:
+    regex: ^master
     tag: preview
-    increment: Minor
\ No newline at end of file
+    increment: patch
```

Comment:
@HowardvanRooijen are you happy to change the default SemVer increment to `patch` rather than `minor`?